Mathlib/AlgebraicGeometry/Morphisms/RingHomProperties.lean

```diff
@@ -114,10 +116,11 @@ theorem sourceAffineLocally_respectsIso (h₁ : RingHom.RespectsIso P) :
       inferInstanceAs (IsIso (e.hom.app _ ≫
         X.presheaf.map (eqToHom (e.hom.preimage_image_eq _).symm).op))
     rw [← Scheme.appLE_comp_appLE _ _ ⊤ (e.hom ''ᵁ U) U.1 le_top (e.hom.preimage_image_eq _).ge,
-      h₁.cancel_right_isIso]
+      CommRingCat.hom_comp, h₁.cancel_right_isIso]
     exact H ⟨_, U.prop.image_of_isOpenImmersion e.hom⟩
   · introv H U
-    rw [Scheme.comp_appLE, h₁.cancel_left_isIso]
+    -- TODO: define a lemma combining `CommRingCat.hom_comp, h₁.cancel_left_isIso`?
+    rw [Scheme.comp_appLE, CommRingCat.hom_comp, h₁.cancel_left_isIso]
```

Comment:
Is that combination worth it or are we now going to have to redefine every concrete categorical result for `RingCat`s?

Reply:
I still believe that we should get rid of the generic `FunLike` instance on `ConcreteCategory` and instead either state everything that currently needs it for `BundledHomCategory` or have `ConcreteCategory` carry an explicit `CoeFun`, so that the generic `hom` and the `<...>Cat.Hom.hom` coincide reducibly.

Reply:
Ah sorry, I did not pay attention well enough here: The lemma that is applied here is `RingHom.RespectsIso.cancel_left_isIso` which is only relevant for `CommRingCat` anyway. So I think this is independent of the concrete category issue. Since `hom_comp` is `@[simp]`, I think we don't need a second version here.